tests/test_parse_metadata.py

```diff
@@ -34,3 +35,28 @@ def test_metadata_for_dwd_data():
     with pytest.raises(requests.exceptions.HTTPError):
         metadata_for_dwd_data(
             Parameter.CLIMATE_SUMMARY, TimeResolution.MINUTE_1, PeriodType.HISTORICAL)
+
+
+def test_metadata_geojson():
+    reset_meta_index_cache()
+
+    # Existing combination of parameters
+    df = metadata_for_dwd_data(
+        Parameter.CLIMATE_SUMMARY, TimeResolution.DAILY, PeriodType.HISTORICAL)
+
+    assert not df.empty
+
+    df = df[df.STATION_ID == 1]
```

Comment:
I tend to use the Column enumeration instead for it to be waterproof in case we might change the column naming later on. However this is also changeable for future cases if the test fails.

Reply:
I've amended the line accordingly. Thanks!